packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -524,7 +524,7 @@ export const Webchat = forwardRef((props, ref) => {
     if (isMedia(input)) input.data = await readDataURL(input.data)
     sendUserInput(input)
     updateLatestInput(input)
-    updateLastMessageDate(new Date())
+    isOnline && updateLastMessageDate(new Date().toISOString())
```

Comment:
do we need to add updateLastMessageDate to somewhere in index.d.ts?

Reply:
added 👍